### ISSUE TITLE
Quest "The missing Diplomat" ( fighting private Hendel part ) flagging for PVP

### ISSUE THREAD

**Issue by Dualityz** [slowtorta/turtlewow-bug-tracker#4111]
Title. The quest "The missing Diplomat" is currently flagging for PVP anyone who's turning the quest in and tries fighting private Hendel. 

**Steps to reproduce the behavior with as much detail as possible:**

1. Turn in https://database.turtle-wow.org/?quest=1266 
2. Accept the followup https://database.turtle-wow.org/?quest=1324
3. PVP Flagged. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: 
https://database.turtle-wow.org/?npc=4966
https://database.turtle-wow.org/?npc=5184 ( two mobs called Theramore Sentry, turning red and attacking you when Private Hendel does so )

Quest: 
https://database.turtle-wow.org/?quest=1266 
https://database.turtle-wow.org/?quest=1324

**Expected behavior. Describe how it should work:**

Should not flag you for PvP.

**Comment by Dalzedur** [comment]
Yeah, and it does that on official too. Hendel is faction-flagged and damaging/debuffing him will flag you for pvp normally. This makes it dangerous for one-lifers to do the quest, since you can easily get ganked after engaging Hendel. There are workarounds, such as running to the nearby watchtower where the guards will beat Hendel for you, so you don't get pvp flagged, but that doesn't change the fact that this probably should not flag you for pvp in an ideal world.

I've only a few times been able to attack Hendel on official and not be pvp flagged, even there it's weird.

**Comment by twowTurtReynolds** [comment]
Tracking under https://github.com/slowtorta/turtlewow-bug-tracker/issues/3935
